Standalone/Steamworks.NET.Standard.csproj

```diff
@@ -1,15 +1,44 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netstandard2.1</TargetFramework>
     <RootNamespace>Steamworks</RootNamespace>
     <AssemblyName>Steamworks.NET</AssemblyName>
     <Platforms>x64;x86</Platforms>
     <Configurations>Windows;OSX-Linux</Configurations>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <RepositoryType>git</RepositoryType>
   </PropertyGroup>
+  
+  <PropertyGroup Label="Nuget PM">
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <PackageId>Steamworks.NET.Standard.$(Configuration).$(platform)</PackageId>
+    <Authors>rlabrecque</Authors>
+    <PackageVersion>14.0.1.2</PackageVersion>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/rlabrecque/Steamworks.NET</PackageProjectUrl>
+    <RepositoryUrl>https://github.com/rlabrecque/Steamworks.NET.git</RepositoryUrl>
+  </PropertyGroup>
+    
+  <ItemGroup>
+    <Compile Remove="%2a%2a\**" />
```

Comment:
This seems dodgy... mixing URL-escaped characters with unescaped ones?
AFAIK they should all be unescaped.

Reply:
I don't know why, remove or edit this will cause complication failure. So I deceided to keep it.